var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -33,18 +33,29 @@ class Sundials(CMakePackage):
 
     homepage = "https://computation.llnl.gov/projects/sundials"
     url = "https://computation.llnl.gov/projects/sundials/download/sundials-2.7.0.tar.gz"
-    maintainers = ['cswoodward', 'gardner48']
+    maintainers = ['cswoodward', 'gardner48', 'balos1']
 
     # ==========================================================================
     # Versions
     # ==========================================================================
 
-    version('4.0.0-dev', '1d4b538721b84ebc91ce7ad92d94beae')
-    version('3.1.1', 'e63f4de0be5be97f750b30b0fa11ef34', preferred=True)
-    version('3.1.0', '1a84ca41c7f71067e03d519ddbcd9dae')
-    version('3.0.0', '5163a44cedd7398bddda442ba00313b8')
-    version('2.7.0', 'c304631b9bc82877d7b0e9f4d4fd94d3')
-    version('2.6.2', '3deeb0ede9f514184c6bd83ecab77d95')
+    version('4.0.0-dev.1',
```

Comment:
Note that you don't have to wrap these -- we add exceptions for directives like `version` and for lines that contain long hashes.

Reply:
Is it preferred to have them unwrapped? If so, I can update the PR.